scripts/build/build.js

```diff
@@ -21,6 +21,7 @@ process.on("unhandledRejection", (err) => {
   process.exit(1);
 });
 
+const CACHE_VERSION = "v22"; // This need update when updating build scripts
```

Comment:
I always have trouble finding this, I know it's somewhere just can't remember, hope moving to top can help.